platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/util/PropertiesFileSupport.java

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
This seems like it should be part of the PropertiesLoader class that already exists.